site/src/pages/RoadmapPage/transfers/GESection.tsx

```diff
@@ -1,8 +1,79 @@
-import { FC } from 'react';
-import MenuSection from './MenuSection';
+import './GESection.scss';
+import { FC, FormEvent, useState } from 'react';
+import MenuSection, { SectionDescription } from './MenuSection';
+
+interface GEInputProps {
+  value: number;
+  setValue: (value: number) => void;
+}
+
+const GEInput: FC<GEInputProps> = ({ value, setValue }) => {
+  const handleSubmit = (event: FormEvent) => {
+    event.preventDefault();
+    const formData = new FormData(event.target as HTMLFormElement);
+    const value = parseFloat(formData.get('value') as string);
+    setValue(value);
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      {/** @todo add restriction for integers only, no floats */}
+      <input className="ge-input" type="number" min="0" name="value" defaultValue={value} />
+    </form>
+  );
+};
+
+interface GETileProps {
+  title: string;
+}
+
+const GETile: FC<GETileProps> = ({ title }) => {
+  {
+    /** @todo figure out how to save this data */
+  }
+  const [numCourses, setNumCourses] = useState<number>(0);
+  const [units, setUnits] = useState<number>(0);
+
+  return (
+    <div className="menu-tile">
+      <div className="tile-info">
+        <p className="name">{title}</p>
+      </div>
```

Comment:
The purpose of having a shared `<MenuTile/>` component is to not rewrite this boilerplate structure. Notice how in `<APExamSection/>`, the tag's children are the contents that show up below the title.

This return should look more like:
```tsx
return <MenuTile title={title}>
  <div className="ge-inputs">...</div>
</MenuTile>
```